brokers/unified/broker.go

```diff
@@ -331,12 +331,14 @@ func resolveRelativeExtensionPaths(metas []model.PluginMeta, defaultRegistry str
 // to be passed back to Che.
 func ConvertMetaToPlugin(meta model.PluginMeta) model.ChePlugin {
 	return model.ChePlugin{
-		ID:           meta.ID,
-		Name:         meta.Name,
-		Publisher:    meta.Publisher,
-		Version:      meta.Version,
-		Containers:   meta.Spec.Containers,
-		Endpoints:    meta.Spec.Endpoints,
-		WorkspaceEnv: meta.Spec.WorkspaceEnv,
+		ID:            meta.ID,
+		Name:          meta.Name,
+		Type:          meta.Type,
```

Comment:
So I assume adding the type here is only a way to forward the type (defined in the meta.yaml) to the Che server in order to use it in the Che server Kubernetes infrastructure ?